### PR TITLE
Add API configuration

### PR DESCRIPTION
**Pull request by marilirulita**
Add files configuration for creating a game using Leaderboard API.
Create a GET request function and POST request function to save and display users' names and scores.

### CODE REVIEW

File: src/new-score.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      user: name,
+      score: scr,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  };
+  const request = new Request(requestURL);
+  await fetch(request, method);
+}
```
Comment (Lameck1) [inline src/new-score.js]: - [ ] In this project, and going forward we are required to use ES6. You must have come across this in milestone one in the learning objectives. Please let's refactor all functions to be of ES6 syntax.

File: src/score-display.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-export { displayScores as default };
\ No newline at end of file
+async function getAllScores() {
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0CasFsHb7LOJqeFKBD5E/scores/';
+  const request = new Request(requestURL);
+  const response = await fetch(request);
+  const players = await response.json();
+
+  displayScores(players);
+}
```
Comment (Lameck1) [inline src/score-display.js]: - The same issue as above, let's refactor accordingly.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @marilirulita, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @marilirulita ,  

Your project is complete! Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**